policies/push-global.rego

```diff
@@ -89,5 +89,6 @@ deps := [dep | startswith(labels[i], "deps:"); dep := split(labels[i], ":")[1]]
 
 # Check if any of the stack dependencies have been modified
 stack_config_affected {
+    startswith(affected_files[_], deps[_])
```

Comment:
this is not required, we test if the affected file names end with the `deps` in the labels

Reply:
@aknysh Yeah, two things: 

1. I want `startswith` as well as `endswith` to support watching directories. e.g. `deps:config/` as that will help me with triggering when any changes in that directory change without having to list out all files in that directory. Does that make sense. 
1. I just learned that this won't work because rego won't do an `or` of these conditions and it's an explicit `and`, so I will update for that.

Reply:
yep, it's an `AND`.
`OR` is done by adding the same block with diff condition

Reply:
Yeah just realized that 😅  Updated in most recent commit 👍